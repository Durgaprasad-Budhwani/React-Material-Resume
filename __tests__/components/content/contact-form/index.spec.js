/**
 * Created by Durgaprasad Budhwani on 10/10/2017.
 */
/* global expect,it */
import React from 'react';
import ContactForm from '../../../../src/components/content/contact-form';
import renderer from 'react-test-renderer';

it('renders contact form', () => {
    const tree = renderer.create(
        <ContactForm />
    ).toJSON();
    expect(tree).toMatchSnapshot();
});
