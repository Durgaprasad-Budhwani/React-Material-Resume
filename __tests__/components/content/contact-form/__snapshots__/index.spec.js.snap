// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders contact form 1`] = `
<div
  className="MuiPaper-root-3 MuiPaper-elevation2-7 MuiPaper-rounded-4 ContactForm-container-1"
>
  <form
    action="https://formspree.io/durgaprasad.budhwani@gmail.com."
    method="post"
  >
    <div
      className="MuiGrid-container-30 MuiGrid-direction-xs-column-33 MuiGrid-justify-xs-center-47"
    >
      <h2
        style={
          Object {
            "marginBottom": 0,
          }
        }
      >
        Feel free to contact me
      </h2>
      <div
        className="MuiFormControl-root-127 MuiFormControl-marginNormal-128"
      >
        <label
          className="MuiFormLabel-root-142 MuiFormLabel-required-147 MuiInputLabel-required-135 MuiInputLabel-root-131 MuiInputLabel-formControl-136 MuiInputLabel-animated-139"
          data-shrink={false}
          htmlFor="name"
        >
          NAME
          <span
            className="MuiFormLabel-asterisk-148"
          >
             *
          </span>
        </label>
        <div
          className="MuiInputBase-root-162 MuiInput-root-149 MuiInput-underline-153 MuiInputBase-formControl-163 MuiInput-formControl-150"
          onClick={[Function]}
        >
          <input
            aria-invalid={false}
            className="MuiInputBase-input-172 MuiInput-input-157"
            disabled={false}
            id="name"
            name="name"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            required={true}
            type="text"
          />
        </div>
      </div>
      <div
        className="MuiFormControl-root-127 MuiFormControl-marginNormal-128"
      >
        <label
          className="MuiFormLabel-root-142 MuiFormLabel-required-147 MuiInputLabel-required-135 MuiInputLabel-root-131 MuiInputLabel-formControl-136 MuiInputLabel-animated-139"
          data-shrink={false}
          htmlFor="email"
        >
          EMAIL
          <span
            className="MuiFormLabel-asterisk-148"
          >
             *
          </span>
        </label>
        <div
          className="MuiInputBase-root-162 MuiInput-root-149 MuiInput-underline-153 MuiInputBase-formControl-163 MuiInput-formControl-150"
          onClick={[Function]}
        >
          <input
            aria-invalid={false}
            className="MuiInputBase-input-172 MuiInput-input-157 MuiInputBase-inputType-175 MuiInput-inputType-160"
            disabled={false}
            id="email"
            name="email"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            required={true}
            type="email"
          />
        </div>
      </div>
      <div
        className="MuiFormControl-root-127 MuiFormControl-marginNormal-128"
      >
        <label
          className="MuiFormLabel-root-142 MuiFormLabel-required-147 MuiInputLabel-required-135 MuiInputLabel-root-131 MuiInputLabel-formControl-136 MuiInputLabel-animated-139"
          data-shrink={false}
          htmlFor="subject"
        >
          SUBJECT
          <span
            className="MuiFormLabel-asterisk-148"
          >
             *
          </span>
        </label>
        <div
          className="MuiInputBase-root-162 MuiInput-root-149 MuiInput-underline-153 MuiInputBase-formControl-163 MuiInput-formControl-150"
          onClick={[Function]}
        >
          <input
            aria-invalid={false}
            className="MuiInputBase-input-172 MuiInput-input-157"
            disabled={false}
            id="subject"
            name="subject"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            required={true}
            type="text"
          />
        </div>
      </div>
      <div
        className="MuiFormControl-root-127 MuiFormControl-marginNormal-128"
      >
        <label
          className="MuiFormLabel-root-142 MuiFormLabel-required-147 MuiInputLabel-required-135 MuiInputLabel-root-131 MuiInputLabel-formControl-136 MuiInputLabel-animated-139"
          data-shrink={false}
          htmlFor="message"
        >
          MESSAGE
          <span
            className="MuiFormLabel-asterisk-148"
          >
             *
          </span>
        </label>
        <div
          className="MuiInputBase-root-162 MuiInput-root-149 MuiInput-underline-153 MuiInputBase-formControl-163 MuiInput-formControl-150 MuiInputBase-multiline-170 MuiInput-multiline-155"
          onClick={[Function]}
        >
          <div
            className="MuiPrivateTextarea-root-179"
          >
            <textarea
              aria-hidden="true"
              className="MuiPrivateTextarea-textarea-180 MuiPrivateTextarea-shadow-181"
              readOnly={true}
              rows="1"
              tabIndex={-1}
              value=""
            />
            <textarea
              aria-hidden="true"
              className="MuiPrivateTextarea-textarea-180 MuiPrivateTextarea-shadow-181"
              readOnly={true}
              rows={1}
              tabIndex={-1}
            />
            <textarea
              aria-invalid={false}
              className="MuiPrivateTextarea-textarea-180 MuiInputBase-input-172 MuiInput-input-157 MuiInputBase-inputMultiline-174 MuiInput-inputMultiline-159"
              disabled={false}
              id="message"
              name="message"
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              required={true}
              rows={1}
              style={
                Object {
                  "height": 19,
                }
              }
            />
          </div>
        </div>
      </div>
      <div>
        <button
          className="MuiButtonBase-root-208 MuiButton-root-182 MuiButton-contained-193 MuiButton-containedPrimary-194 MuiButton-raised-196 MuiButton-raisedPrimary-197 ContactForm-button-2"
          disabled={false}
          onBlur={[Function]}
          onContextMenu={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex="0"
          type="submit"
        >
          <span
            className="MuiButton-label-183"
          >
            SEND
          </span>
          <span
            className="MuiTouchRipple-root-211"
          />
        </button>
      </div>
    </div>
  </form>
</div>
`;
